src/packagedcode/maven.py

```diff
@@ -1026,6 +1025,21 @@ def parse(location=None, text=None, check_is_pom=True, extra_properties=None):
     return package
 
 
+def get_maven_licenses(licenses):
+    """
+    Return a list of order dictionary to store license by paring the passing maven licenses
+    """
+    declared_license = []
+    if licenses:
+        for each in licenses:
+            declared_license.append(OrderedDict([
+                ('name', each.get('name')),
+                ('url', each.get('url')),
+                ('comments', each.get('comments')),
+                ('distribution', each.get('comments')),
```

Comment:
Do we care about the distribution? And this is unlikely the comments, right? What are the typical values?

Reply:
Yep, agree, removed it.``